tools/dataProtector/getting-started.md

```diff
@@ -38,9 +38,7 @@ bun add @iexec/dataprotector
 **This package is an ESM package. Your project needs to use ESM too.**
 &nbsp;[**Read more**](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c)
 
-If you use it with **Webpack**, some polyfills will be needed. You can find a
-minimal working project
-[here](https://github.com/iExecBlockchainComputing/dataprotector-sdk/tree/main/demo/browser-webpack).
+If you use it with **Webpack**, some polyfills will be needed.
```

Comment:
why removing link to the demo ?

Reply:
like this permalink : https://github.com/iExecBlockchainComputing/dataprotector-sdk/tree/115b797cf62dcff0f41e2ba783405d5083d78922/packages/demo/browser-webpack